app/src/main/java/nl/tudelft/bbw/crawler/BlockWriter.java

```diff
@@ -0,0 +1,43 @@
+package nl.tudelft.bbw.crawler;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.Map;
+
+/**
+ * Class to write a block to a file
+ */
+final class BlockWriter {
+
+    /**
+     * Class attributes
+     */
+    public static final String OUTPUT_PATH = "output.json";
```

Comment:
I understand, but why would you not give the path as an argument? This way it is not very clear where the data goes when writing